crates/aptos-build-info/src/lib.rs

```diff
@@ -54,6 +55,9 @@ pub fn get_build_information() -> BTreeMap<String, String> {
     build_information.insert(BUILD_RUST_CHANNEL.into(), build::RUST_CHANNEL.into());
     build_information.insert(BUILD_RUST_VERSION.into(), build::RUST_VERSION.into());
 
+    // Compilation information
+    build_information.insert(BUILD_IS_RELEASE_BUILD.into(), is_release().to_string());
```

Comment:
is there a way to get the build profile name? as opposed to `release` or `debug` ? We're also building `performance` profile these days

Reply:
Still looking into it but it's not looking good. Asking on SO: https://stackoverflow.com/questions/73595435/how-to-get-profile-from-cargo-toml-in-build-rs-or-at-runtime.

Reply:
Okay got it working, with a known set of caveats + explanation.